examples/common/resumption_callbacks/callbacks.py

```diff
@@ -0,0 +1,50 @@
+from composer.core import Callback, State
+from composer.loggers import Logger
+from typing import List
+from abc import ABC
+
+__all__ = ['ResumptionCallback', 'GlobalLRScaling', 'LayerFreezing',]
+
+
+class ResumptionCallback(Callback, ABC):
+    pass
```

Comment:
@bmosaicml are there any plans for this subclass to have specific traits that a standard `Callback` does not have? If there is no difference, then I would prefer to:

* remove `ResumptionCallback` and make all functions below just subclass `Callback`
* rename this file `resumption_callbacks.py` and move it to `examples/common/resumption_callbacks.py`

Reply:
yeah the sounds good to me!